enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/components/SdkIframeEmbedRoute.tsx

```diff
@@ -40,7 +43,9 @@ const store = getSdkStore();
 createTracker(store);
 
 export const SdkIframeEmbedRoute = () => {
-  const { embedSettings } = useSdkIframeEmbedEventBus({ onSettingsChanged });
+  const { embedSettings, usageAnalytics } = useSdkIframeEmbedEventBus({
```

Comment:
We need to report analytics where we can access the redux store because I need to get the setting `site-url`